mmv1/products/alloydb/Cluster.yaml

```diff
@@ -559,3 +559,37 @@ properties:
                   name: nanos
                   description: |
                     Fractions of seconds in nanoseconds. Currently, only the value 0 is supported.
+  - !ruby/object:Api::Type::Enum
+    name: 'subscriptionType'
+    values:
+      - :TRIAL
+      - :STANDARD
+    default_from_api: true
+    description: |
+      The subscrition type of cluster.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'trialMetadata'
+    description: |
+      Contains information and all metadata related to TRIAL clusters.
+    output: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: startTime
+        description: |
+          Start time of the trial cluster.
+        default_from_api: true
```

Comment:
Please remove default_from_api here and in all other fields within this object. output: true is inherited, but it will conflict with default_from_api: true

Reply:
Done. Thanks!